### PR TITLE
Feature: add the hability to listen sidebar/plugins actions

### PR DESCRIPTION
**Pull request by icaromh**

### CODE REVIEW

File: src/components/ActionsProvider.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
```
Comment (marcelometal) [inline src/components/ActionsProvider.js]: Hi @icaromh, Could you please put the copyright holder and license inside this file?

File: src/constants.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const SIDEBAR_EXPAND = "SIDEBAR_EXPAND";
```
Comment (marcelometal) [inline src/constants.js]: Hi @icaromh, Could you please put the copyright holder and license inside this file?

File: src/constants.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2016, Globo.com (https://github.com/globocom)
```
Comment (marcelometal) [inline src/constants.js]: Please, the year is 2019 =)

File: src/components/ActionsProvider.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2016, Globo.com (https://github.com/globocom)
```
Comment (marcelometal) [inline src/components/ActionsProvider.js]: Please, the year is 2019 =)

File: src/constants.js (hunk 3 above)
Comment (icaromh) [reply src/constants.js]: 🤦‍♂️

**Pull request merged**